site/components/navigator/index.js

```diff
@@ -0,0 +1,85 @@
+// @flow
+
+import React from 'react';
+import classnames from 'classnames';
+import styles from './style.css';
+
+type Props = {
+  currentIndex: number,
+  maxIndex: number,
+  onChange: number => void,
+};
+
+function mapOverRange(count, mapper) {
```

Comment:
Could use something like this here:

```js
// arrayFromNumber(5) // [0, 1, 2, 3, 4]
const arrayFromNumber = number => [...Array(number).keys()];

// Usage...
arrayFromNumber(maxIndex + 1).map((_, i) => (
  <Item key={x} value={x} currentIndex={currentIndex} />
));
```

It's arguably more declarative, but probably not as readable. I might also have completely misunderstood the original intent so ignore me if I have 🙈 If you do decide to go with this remember to remove the eslint rule that was added.